src/components/modals/ResendConfirmationEmailModal.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <b-modal
+    ok-title="Send"
+    @hidden="reset"
+    v-model="modalVisible"
+    cancel-variant="outline-primary"
+    id="resendConfirmationEmailModal"
+    title="Resend Confirmation Email"
+    v-on:ok="resendConfirmationEmail"
```

Comment:
```suggestion
    @ok.prevent="resendConfirmationEmail"
```

So you don't have to do `preventDefault` in the handler.

Reply:
Actually, the prevent default isn't even necessary in this context, so I'll just remove it altogether.

FYI I copied the structure of this file from the FaucetModal so that's where the `event.preventDefault()` came from.

Reply:
We should still use `@ok` instead of `v-on:ok` since we are already using that shorthand on line 4 for `@hidden`